README.md

```diff
@@ -202,9 +202,57 @@ ResponseCache::forget(['/some-uri', '/other-uri']);
 
 // Alternatively
 ResponseCache::forget('/some-uri', '/other-uri');
+
+
+// Advanced
+ResponseCache::cacheCleaner()
```

Comment:
In think we can make this more human by renaming some methods.

```
ResponseCache::selectCachedItems()
   ->withPutMethod()
   ->withHeader($headers)
  ->usingSuffix(...)
  ->usingTags(...)
  ->forUrls(...)
  ->forget()
```

This feel less programmy.

Reply:
ok